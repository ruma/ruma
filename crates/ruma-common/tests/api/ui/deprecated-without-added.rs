use ruma_common::{api::Metadata, metadata};

const _: Metadata = metadata! {
    method: GET,
    rate_limited: false,
    authentication: None,
    history: {
        unstable => "/a/path",
        1.1 => deprecated,
    }
};

fn main() {}
