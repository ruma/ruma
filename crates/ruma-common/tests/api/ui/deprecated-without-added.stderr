error: no rules expected the token `=>`
 --> tests/api/ui/deprecated-without-added.rs:7:14
  |
7 |     unstable => "/a/path",
  |              ^^ no rules expected this token in macro call
